airbyte-integrations/connectors/source-jira/source_jira/streams.py

```diff
@@ -203,20 +203,23 @@ def request_params(self, stream_state=None, **kwargs):
         stream_state = stream_state or {}
         params = super().request_params(stream_state=stream_state, **kwargs)
         params["fields"] = ["attachment", "issuelinks", "security", "issuetype", "updated"]
-        if stream_state.get(self.cursor_field):
-            issues_state = pendulum.parse(stream_state.get(self.cursor_field))
+        normalized_cursor_name = ".".join(self.cursor_field)
+        if stream_state.get(normalized_cursor_name):
+            issues_state = pendulum.parse(stream_state.get(normalized_cursor_name))
             issues_state_row = issues_state.strftime("%Y/%m/%d %H:%M")
             params["jql"] = f"updated > '{issues_state_row}'"
         return params
 
     def get_updated_state(self, current_stream_state: MutableMapping[str, Any], latest_record: Mapping[str, Any]) -> Mapping[str, Any]:
-        latest_record_date = pendulum.parse(latest_record.get("fields", {}).get(self.cursor_field))
+        normalized_cursor_name = ".".join(self.cursor_field)
```

Comment:
I'm not sure I understand why do we need this `normalized` name here.

Reply:
same here